concepts-wip/bools/about.md

```diff
@@ -0,0 +1,136 @@
+# Bools
+
+Crystal has a type know as [`Bool`][bools], it is used to represent the values `true` and `false`.
+You can use logical operators (`!`, `||`, `&&`) which can be used to combine Bools and create more complex expressions:
+
+## Logical operators
+
+Crystal has 3 logical operators which are used to combine Bools and make expressions that produce different values.
+
+### And(`&&`)
+
+The [_and_ operator][and] in crystal is represented by `&&` and returns `true` if both values given are `true` otherwise it returns `false`.
+When using the _and_ operator, one Bool be placed on the right side of the `&&` and another one on the left side.
+
+```crystal
+true && true
+# => true
+
+true && false
+# => false
+
+false && false
+# => false
+```
+
+### Or(`||`)
+
+The [_or_ operator][or] in crystal is represented by `||` and returns `true` if **at least one** of values given is `true` if both of the values are `false` then it returns `false`.
+When using the _or_ operator one bool should be placed on the right side of the `||` and another one on the left side.
+
+```crystal
+true || true
+# => true
+
+true || false
+# => true
+
+false || true
+# => true
+
+false || false
+# => false
+```
+
+### Not(`!`)
+
+The _not_ operator in crystal is represented by `!` and returns `true` if the given Bool is `false` and return `false` if `true` is given.
+When using the _not_ operator one Bool should be placed after the operator (`!`).
+
+```crystal
+!true
+# => false
+
+!false
+# => true
+```
+
+## Using parentheses(`()`)
+
+When working with booleans you can use parentheses to decided which Bools to evaluate first.
+The result can differ depending on how the parentheses are used.
+
+In crystal, what is in parentheses is evaluated first.
+After parentheses has been evaluated so are booleans evaluated from left to right.
```

Comment:
I have noticed that you tend to do a lot of small grammar mistakes. Have you considered using some sort of automated grammar checker to help you out?

I sometimes use Grammarly. It's not perfect, and it doesn't understand how programmers talk about programming... but it's better than nothing. Here's an example of how Grammarly could help you improve those two sentences:

<img width="1309" alt="Screenshot 2023-02-25 at 13 15 28" src="https://user-images.githubusercontent.com/7852553/221356574-5cde45d8-5f66-49cb-9745-c9c8908bccaf.png">

It doesn't find all the problems, but it's a good start. Here's how I would fix the grammar even further (but I'm not a linguist or a native speaker either)

```suggestion
In Crystal, what is in parentheses is evaluated first.
After parentheses have been evaluated, the booleans are evaluated from left to right.
```

Reply:
I used Grammarly for my earlier concepts and for writing this message XD. I have a spell checker connected to my ide. But I will use Grammarly from now on(So let me write another commit with those changes), but if you see any grammar mistakes(After the commit), please let me know.